security/vault/Portfile

```diff
@@ -53,3 +53,11 @@ post-extract {
 destroot {
     xinstall -m 755 ${workpath}/bin/${name} ${destroot}${prefix}/bin/${name}
 }
+
+variant ui description {Enable the Web UI} {
+    build.post_args    static-dist dev-ui
+    depends_lib-append port:nodejs10 \
```

Comment:
There are many nodejs ports including a number with a higher version. Why not use the latest version (nodejs13) or allow any version?

Reply:
The ui build process specifically requires nodejs 10.x - https://github.com/hashicorp/vault/blob/efa74497e070c643a0507ab4e29c317cce8011bd/ui/package.json#L167